From 92461b725de6d395e4cebcfdfc77fecb6bd7fe21 Mon Sep 17 00:00:00 2001
From: MisterZtr <andey2004586@gmail.com>
Date: Sat Jul 22 16:15:38 2023 +0300
Subject: [PATCH] Remove-duplicated-persist.sys.usb.config

diff --git a/config/common.mk b/config/common.mk
index 5d6e8cb2..9552023a 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -35,7 +35,6 @@ ifeq ($(TARGET_BUILD_VARIANT),user)
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
 else
 # Disable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.usb.config=mtp,adb
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.debuggable=1
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.force.debuggable=true


